.env

```diff
@@ -0,0 +1,6 @@
+#Copy this file to a file named .env and replace with appropriate values.
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=us-east-1
+AWS_SESSION_TOKEN=FwoGZXIvYXdzEPf//////////wEaDJHpsbXYBbogut45kyKrAWIl1iJVMq6wiBCx/1n5gcNDKe9vG3ipreLFKowcwMUve2ANeOvwxo3I/0tvqoOk1IVJw3CrRgVyD9KOZ1hpc7DdbjhqKxDDwwQV6hDiZmU7t+B/HLR+QO92/T9zpnzBcbnkiGag4B4G8IikhxNmgnCqg6fRszy0qAKipuShHhrAzJe8kvLnxDHL2yLSN8AiOC4k2Nc1RPz1vYPWHovjGh3vG2nyE1tdwo2CpyjOu6WEBjIt2+a6KmWAWLYGzwNwNf7C4kj+rHEyBufYgZfiVxrdGBbE8B9aZf5MEHWcjFLh
```

Comment:
Why is this file commited ? It should be in git ignore

Reply:
Issue is resolved